packages/dds/tree/src/test/shared-tree/fuzz/fuzzEditGenerators.ts

```diff
@@ -333,7 +338,31 @@ export const makeTreeEditGenerator = (
 					dstIndex: random.integer(0, field.length),
 				};
 			},
-			weights.move,
+			weights.intraFieldMove,
+			({ fieldInfo }) => fieldInfo.content.length > 0,
+		],
+		[
+			(state): CrossFieldMove => {
+				const srcField = state.fieldInfo.content;
```

Comment:
Are cross-field moves out of optional fields into sequence ones supported? I was under the impression they weren't, but it doesn't seem forbidden here at the generator level.

Reply:
They are indeed not supported. `state.fieldInfo.content` is guaranteed to be a sequence field here.

Reply:
ah, yeah it's going through `sequenceFieldEditGenerator`. Seems fine.